content/en/docs/11/_index.md

```diff
@@ -41,6 +41,16 @@ RUN apt-get update && \
     apt-get clean
 ```
 
+{{% alert title="Note" color="primary" %}}
+Copy the content above. Switch to the vim editor and use the following commands to insert and save everything:
+
+* "i"
+* "crtl-v"
+* "esc"
+* ":wq"
+* "enter"
+{{% /alert %}}
+
```

Comment:
We already tell participants to use their editor of choice (line 35), so explaining how to use `vim` seems a bit off here.

Reply:
I was just wondering finding the "vim Dockerfile" command. Instead we should write then "open a editor of your choice"???

Reply:
That's true, that's misleading too. Maybe we should just remove the command and merge lines 29 and 35 saying something like:

```
Add the following content to the Dockerfile using your editor of choice:
```